app/models/ecf_participant_eligibility.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ECFParticipantEligibility < ApplicationRecord
+  has_paper_trail
+
+  belongs_to :participant_profile, class_name: "ParticipantProfile::ECF", touch: true
+  before_validation :determine_status, on: :create
+
+  enum status: {
+    eligible: "eligible",
+    matched: "matched",
+    manual_check: "manual_check",
+  }, _suffix: true
+
+  def determine_status
+    self.status = if active_flags? || previous_participation? || previous_induction?
+                    :manual_check
+                  elsif !qts?
+                    :matched
+                  else
+                    # NOTE: this should be :eligible here but we are putting
+                    # everyone in the holding pen until we have the data to validate further
+                    :matched
+                  end
```

Comment:
When do we change it?

Reply:
Manually via the console or from an Admin UI or with a code change when we can access the extra data AFAIK